python/vyos/vpp/__init__.py

```diff
@@ -18,3 +18,7 @@
 from .control_vpp import VPPControl
 
 __all__ = ['VPPControl']
+
+
+class VppNotRunningError(Exception):
```

Comment:
Should there be a hierarchy of VPP exceptions now? I'll need to check what else we can add there.